DNN Platform/Library/Entities/Modules/ModuleController.cs

```diff
@@ -1902,6 +1902,21 @@ public void MoveModule(int moduleId, int fromTabId, int toTabId, string toPaneNa
             //Move the module to the Tab
             dataProvider.MoveTabModule(fromTabId, moduleId, toTabId, toPaneName, UserController.Instance.GetCurrentUserInfo().UserID);
 
+            //Update the Tab reference for the module's ContentItems
+            var contentController = Util.GetContentController();
+            var contentItems = contentController.GetContentItemsByModuleId(moduleId);
+            if (contentItems?.Count() > 0)

```

Comment:
Do we need to count first since we are just entering a foreach, if there is no items it just never enters the foreach right?

Reply:
Done.